src/main/scala/allawala/chassis/util/DataGenerator.scala

```diff
@@ -25,11 +25,57 @@ class DataGenerator {
 
   // From com.softwaremill.session.SessionUtil
   def toHexString(array: Array[Byte]): String = {
-    DatatypeConverter.printHexBinary(array)
+    printHexBinary(array)
   }
 
   // From com.softwaremill.session.SessionUtil
   def hexStringToByte(hexString: String): Array[Byte] = {
-    DatatypeConverter.parseHexBinary(hexString)
+    parseHexBinary(hexString)
   }
+
+  // copied from javax.xml.bind.DatatypeConverterImpl, since java.xml.bind is no longer available in java 11
```

Comment:
not sure if this is the best solution

i would prefer if we could import libraries like mentioned 

https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception

they give a few different options for importing